src/arkode/arkode_splittingstep_coefficients.c

```diff
@@ -14,8 +14,10 @@
  * This is the implementation file for splitting coefficients
  *--------------------------------------------------------------*/
 
-#include <arkode/arkode_splittingstep.h>
 #include <stdlib.h>
+#include <string.h>
+
+#include <arkode/arkode_splittingstep.h>
```

Comment:
Is some tool being used to check for missing and unused includes? If so it would be nice have it in scripts/CI (for a separate, future PR of course)

Reply:
My clangd sometimes does this header cleanup if I forget to turn it off.